info/static/info/css/base.js

```diff
@@ -0,0 +1,7 @@
+
+
+$(function(){$(".nav-link").on("click", function(){
```

Comment:
This JS will not affect the change needed as the page reload will happen and this will do nothing meaningful and should be removed.